.github/workflows/Build Thunder on Linux.yml

```diff
@@ -100,6 +100,7 @@ jobs:
       run: |
         cmake -G Ninja -S Thunder -B ${{matrix.build_type}}/build/Thunder \
         -DCMAKE_CXX_FLAGS="-Wall -Wextra -Wpedantic -Werror" \
+        -DBUILD_SHARED_LIBS=ON \
```

Comment:
So just to make sure - do we want actions in all of our other repos to build with Thunder with shared libs enabled?